NEWS.md

```diff
@@ -1,73 +1,62 @@
- * `add-contextual-data()`: Allow using templates in name-value pairs (#2711)
- * Add support for floating point operations in template functions (#2742)
- * Add support for usec precision when parsing time (#2709)
+ * `file(), pipe()`: The `time-reap()` option now can be set or disabled for
+   each destination separately.
+   Use `time-reap(0)` to disable closing idle destination files. (#2798)
+
+ * `syslog-ng-ctl`: Add `list-files` subcommand to print files present in the
+   current configuration. (#2797)
 
 ## Bugfixes
 
- * Fix null pointer access when destinations are suspended (#2778)
- * Fix `grouping-by()` deadlock (#2758)
- * Fix a general source-related crash and enhance `wildcard-file()`'s bookmark
-   handling (#2589)
- * Fix infinite loop (reload/reopen) (#2739)
- * Fix `python()` package/module name collision (#2438)
- * Fix escaped quote in block argument (#2781)
- * Reintroduce test on SYSLOG_NG_HAVE_TIMEZONE (#2774)
- * `snmp()`: Fix template leak (#2746)
+ * Fix minor memory leaks (#2868)
+ * Add global context to standalone parsers (#2876)
+ * Fix heap usage after free in cfg_run_parser_with_main_context (#2884)
+ * Fix g_thread_init call order issue with older glibs (#2853)
+ * SNMP destination: fixing statistics format (#2854)
+ * eventlog: flush escaped_buffer when full (#2837)
+ * dbparser: remove unnecessary lock (#2838)
+ * dbparser: fix crash when context times out in the middle of another rule (#2832)
+ * radix: fix grouping in PCRE (#2808)
+ * add-contextual-data: make filters config plugin aware (#2886)
+ * Fix distro packages build (#2883)
+ * Undefined warning regression (#2829)
 
 ## Other changes
 
- * Never drop flow-controlled messages: The meaning of `log-fifo-size()` has
-   changed to avoid dropping flow-controlled messages when `log-fifo-size()` is
-   misconfigured. From now on, `log-fifo-size()` only affects messages that are
-   not flow-controlled. (#2753)
-
- * The `-d`/`--debug` syslog-ng command line flag no longer implies
-   `-e`/`--stderr`. If you want to redirect `internal()` source to stderr,
-   use the `-e`/`--stderr` option explicitly. (#2731)
-
- * dbld, RPM and DEB packaging improvements (#2724)
- * Checkpoint parser improvements (#2740)
- * Reset the timezone on config reload event (#2691)
- * `geoip2()`: Include IP into the error message (#2743)
- * Improve regexp error messages (#2796)
- * `http()`: Warn if less workers used than urls (#2757)
- * `http()`: Allow URLs to be specified by a space/comma separated string
-   (#2699)
- * loggen: Change message rate at runtime using signals (#2756)
- * debun: add acquire_running_syslog_config function (#2752)
- * FreeBSD fixes for the test suite (#2783)
+ * Disable `time-reap()` on non-templated filenames by default (#2798)
+ * The `--preprocess-into` command line flag accepts `-`, and writes the
+   preprocessed configuration to stdout. (#2869)
+ * Remove POSIX regex support (part 2) (#2849)
```

Comment:
Also, no need to announce this, 
especially with a naming that suggests that further POSIX regex support has been removed.

Reply:
Thanks.